pkg/capabilities/datastreams/types.go

```diff
@@ -79,6 +79,17 @@ type StreamsTriggerEvent struct {
 	Timestamp int64
 }
 
+type LLOStreamsTriggerEvent struct {
+	Payload              []*LLOStreamDecimal
+	ObservationTimestamp int64
```

Comment:
Can we please change to `ObservationTimestampNanoseconds` for compatibility with a future release of LLO which will support sub-second timestamps?
```suggestion
	ObservationTimestampNanoseconds uint64
```